config/application.rb

```diff
@@ -16,5 +16,6 @@ class Application < Rails::Application
                          { expires_in: 90.minutes }
     config.active_job.queue_adapter = :delayed_job
     config.active_record.schema_format = :ruby
+    config.autoload_paths += %W[#{config.root}/app/lib/result_handler]
```

Comment:
Don't forget about `eager_load_paths`, because [in some env's you can face problems](https://blog.arkency.com/2014/11/dont-forget-about-eager-load-when-extending-autoload/) with Rails Autoloading.